### PR TITLE
Remove Travis [testing/bernede1/disable-travis]

### PR DESCRIPTION
**Pull request by adrienbernede**
This branch removes travis configuration and updates the CONTRIBUTING.md and CHANGELOG.md accordingly.

Next:
- [x] Remove Travis from the required checks (@tzanio or @v-dobrev)
- [x] Add Github Actions to the required checks  (@tzanio or @v-dobrev)

Note:
It is also possible to set gitlab as a required check.

 | PR | Author | Editor | Reviewers | Assignment | Approval | Merge| 
 | --- | --- | --- | --- | --- | --- | --- | 
| [#2397](https://github.com/mfem/mfem/pull/2397) | @adrienbernede | @tzanio | @tzanio + @acfisher | 07/09/21 | 07/16/21 | 07/18/21 | |

### CODE REVIEW

**Comment by tzanio** [comment]
This PR is now [under review](https://github.com/mfem/mfem/blob/master/CONTRIBUTING.md#pull-requests) (see the table in the PR description). To help with the review process, please **do not force push** to the branch.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,6 @@
 
 <p align="center">
 <a href="https://github.com/mfem/mfem/blob/master/LICENSE"><img alt="License" src="https://img.shields.io/badge/License-BSD-brightgreen.svg"></a>
-<a href="https://travis-ci.org/mfem/mfem"><img alt="Build Status" src="https://travis-ci.org/mfem/mfem.svg?branch=master"></a>
```
Comment (tzanio) [inline CONTRIBUTING.md:7]: Is there an equivalent GH actions badge?

**Review by tzanio — APPROVED** [review]
Sad to see it go, but we have to move on...

**Review by acfisher — APPROVED** [review]
Goodbye Travis.  It was fun while it lasted.

**Comment by tzanio** [comment]
Merged in `next` for testing...

File: CONTRIBUTING.md (hunk 1 above)
Comment (adrienbernede) [reply CONTRIBUTING.md:7]: @tzanio I had missed that, so I created https://github.com/mfem/mfem/pull/2414.

**Pull request merged**